woocommerce/woocommerce-android
# Issue/6924 tracking banner dismiss event

Closes: #6924 

### Description
This PR adds tracking when the upsell card reader banner is dismissed on the payments, order list, and settings screen.

event name: `woocommerceandroid_feature_card_dismissed`

properties:

**For Payments screen:**
source. payment_methods
campaign_name. In this case, the value will be upsell_card_readers
remind_later – boolean to track whether the user requested to be reminded later or not.

**For the Order list screen:**
source. order_list
campaign_name. In this case, the value will be upsell_card_readers
remind_later – boolean to track whether the user requested to be reminded later or not.

**For the settings screen:**
source. settings
campaign_name. In this case, the value will be upsell_card_readers
remind_later – boolean to track whether the user requested to be reminded later or not.

### Testing instructions

Tracking in the Payments screen
1. Navigate to the order eligible for IPP
2. Notice the Banner
3. Tap on different CTAs and ensure the events are being tracked in the logs

Tracking in the Order list screen
1. Navigate to the order list screen
2. Notice the Banner
3. Tap on different CTAs and ensure the events are being tracked in the logs

Tracking in the Settings screen
1. Navigate to the settings screen
2. Notice the Banner
3. Tap on different CTAs and ensure the events are being tracked in the logs
